website/versioned_docs/version-v1.14.0/deployment/distribution_component.md

```diff
@@ -49,7 +49,7 @@ Credentials Endpoint with a configuration targeted at Cumulus and NGAP.
 
 ```hcl
 module "distribution" {
-  source = "https://github.com/nasa/cumulus/archive/terraform-aws-cumulus-distribution-1.13.1.zip"
+  source = "https://github.com/nasa/cumulus/archive/terraform-aws-tea-distribution-1.13.1.zip"
```

Comment:
why are we renaming these?  Are we going to create `terraform-aws-tea-distribution-1.13.1.zip`?

Reply:
Oh yeah, good question. So my thinking here was that I wanted to replace it with the new package name, `terraform-aws-tea-distribution` to try and avoid confusion. This is pointing to a specific 1.13.1 version of the .zip which doesn't exist but I was thinking this was part of the release process to ensure that the current version is reflected here.

I could 100% be wrong though. That was an assumption I didn't realize I was making.

Reply:
I think we don't need to update the doc for previous releases, `terraform-aws-cumulus-distribution-1.13.1.zip` was correct for those releases.
deployment/distribution_component.md no longer exists for current release.
Instead, you can refer to the new release name and version under `tf-modules/distribution/README.md`

Reply:
Oh, ok. I see what you're saying. I was thinking we would need to scrub all references to the previous file names but yes, if someone is on a (much) earlier version they would have the original .zip files with the old names.

I will not re-write history :-)
https://github.com/nasa/cumulus/pull/2279/commits/8240eb02affd3de4f47b714d278ee90330d5391a

In the above I made the release version a generic x.x.x in the distribution/README.md as an example but actually the original link there for https://github.com/nasa/cumulus/archive/terraform-aws-tea-distribution-1.13.1.zip isn't valid. Should this be a link to the most current release's artifact?

https://github.com/nasa/cumulus/releases/download/v9.1.0/terraform-aws-cumulus-distribution.zip 

Again, it's an example so maybe it doesn't totally matter but getting it close would probably be good.